### DISCUSSION TITLE
Connects, but can't read files

### DISCUSSION THREAD

**Discussion by FreshImmuc** [QuorumDMS/ftp-srv#328] [Q&A]
First I see this:
![image](https://user-images.githubusercontent.com/65733898/175400283-e041a9d5-3371-4a35-8b8c-2eb3d1ba00d8.png)

And directly after that it shows this error message:
![image](https://user-images.githubusercontent.com/65733898/175400341-62069588-b4c6-4d8f-a0da-79d979113cf2.png)

What can I do to fix that? What could throw the error?

**Comment by matt-forster** [comment]
It depends on how you are managing your filesystem - it is hard to help this one without more information on how you are registering your FS and serving it to your commands.
